app/controllers/reminders/edit.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    updateReminder() {
+      this.transitionToRoute('reminders')
```

Comment:
So here you're not saving the changes of your model, so they most likely have Dirty attributes. You should bring the model in here and save the new changes.  Semi-colons :)